docs/install.md

```diff
@@ -36,23 +37,25 @@ If you want to try the latest developing Kube-OVN, try the script below
 2. Use vim to edit the script variables to meet your requirement
 ```bash
  REGISTRY="kubeovn"
- POD_CIDR="10.16.0.0/16"                # Do NOT overlap with NODE/SVC/JOIN CIDR
- SVC_CIDR="10.96.0.0/12"                # Do NOT overlap with NODE/POD/JOIN CIDR
- JOIN_CIDR="100.64.0.0/16"              # Do NOT overlap with NODE/POD/SVC CIDR
+ POD_CIDR="10.16.0.0/16"                # Default subnet cidr, Do NOT overlap with NODE/SVC/JOIN CIDR
+ SVC_CIDR="10.96.0.0/12"                # Should be equal with service-cluster-ip-range CIDR range that is configured for the API server
+ JOIN_CIDR="100.64.0.0/16"              # Subnet cidr that connect node to pod, Do NOT overlap with NODE/POD/SVC CIDR
  LABEL="node-role.kubernetes.io/master" # The node label to deploy OVN DB
- IFACE=""                               # The nic to support container network can be a nic name or a group of regex separated by comma, if empty will use the nic that the default route use
- VERSION="v1.7.0"
+ IFACE=""                               # The nic to support container network can be a nic name or a group of regex separated by comma e.g. `IFACE=enp6s0f0,eth.*`, if empty will use the nic that the default route use
+ VERSION="v1.7.1"
 ```
 
-After v1.6.0 `IFACE` support regex, e.g. `IFACE=enp6s0f0,eth.*`
+This basic setup works for default overlay network, for default underlay/vlan network, please check [here](vlan.md)
 
-3. Execute the script
+3. Run the script
 
 `bash install.sh`
 
+That's all! You can now create some pods and test connectivity.
+
 ### Step by Step Install
 
-If you want to know the detail steps to install Kube-OVN, please follow the steps.
+The one script installer is the recommended way to install Kube-OVN. If you want to change the default options when install Kube-OVN, please follow the steps.
```

Comment:
The one-script installer is recommended. If you want to change the default options, follow the steps below.